shell/common/v8_value_serializer.cc

```diff
@@ -20,7 +20,9 @@ const uint8_t kVersionTag = 0xFF;
 class V8Serializer : public v8::ValueSerializer::Delegate {
  public:
   explicit V8Serializer(v8::Isolate* isolate)
-      : isolate_(isolate), serializer_(isolate, this) {}
+      : isolate_(isolate),
+        serializer_(isolate, this),
+        microtasks_scope_(isolate, v8::MicrotasksScope::kDoNotRunMicrotasks) {}
```

Comment:
It doesn't seem right to put the microtasks scope at the class level here, perhaps it should go in `Serialize`?

Reply:
ok, I'll change it